Update README.md (#26)
nglgzz/awesome-clean-tech: README.md

- [Vestas](https://www.vestas.com/) | Denmark | 1945

  Manufacturer, seller, installer, and servicer of wind turbines. As of 2019 Vestas has installed over 66,000 wind turbines for a capacity of 100 GW in over 80 countries on five continents.
  
- [Suzlon Energy Ltd.](https://www.suzlon.com/) | India | 1995

> Suzlon is a win turbine manufacturer. It is present in 18 countries across the continents.

Reviewer: ```suggestion
Wind turbine manufacturer present in 18 countries across the continents.
```

- [Siemens](https://new.siemens.com/in/en.html) | Germany | 1847

> It manufactures wind turbines and provides onshore and offshore wind services. It is the world's second largest wind turbine manufacturer.

Reviewer: According to https://www.statista.com/statistics/271031/market-share-of-the-largest-wind-turbine-manufacturers-in-the-world/

```suggestion
Multinational manufacturing company. Among their many projects they produce wind turbines and provide onshore and offshore wind services. It is the world's third largest wind turbine manufacturer as of 2018.
```
